crates/types/src/calibration.rs

```diff
@@ -0,0 +1,42 @@
+use coordinate_systems::Ground;
+use linear_algebra::Point2;
+use path_serde::{PathDeserialize, PathIntrospect, PathSerialize};
+use serde::{Deserialize, Serialize};
+
+use crate::{camera_position::CameraPosition, cycle_time::CycleTime};
+
+#[derive(
+    Clone, Debug, Default, Serialize, Deserialize, PathSerialize, PathDeserialize, PathIntrospect,
+)]
+pub enum CalibrationCommand {
```

Comment:
I agree with you, and have the same concern. This was simply repeated at the WIP state to make things easy to test at the beginning. And I needed to verify that these states were enough for the job.

Now, the question would be how to avoid repitition of defining `LookAt` and `Capture` in the internal state and output command. I'll think of some ideas and try to simplify this.

Reply:
I think that is a small amount of duplication to pay in exchange for a much better separation of states and commands.
Don't overthink it 😉

Reply:
I fixed this, but there are a couple of things I'm not sure in terms of style:
1. the internal state enum is defined in the controller's module (no point to have it in a different crate)
2. IDK whether to keep the definition it at end of the file, top of the file or just before the `impl ... ` for the structs (what I've done now)